sites/all/modules/reol_carousel/js/reol_carousel.js

```diff
@@ -230,7 +229,7 @@
           (tab.data('offset') > -1 &&
           // Use .75 to make it trigger when scrolling to the end when the
           // coursel only has 5 elements as it will have when initially loaded.
-          this.progress > 0.75)) {
+          this.progress > 0.60)) {
```

Comment:
The value `0.60` does not match the comment above it. I don't understand the comment. Can we say something clever about why the value must be `0.60`? Otherwise, we'll just let it be as it is now.

Reply:
I played around with this value, and found it solved a issue with covers not aligning on the left side of the container.
Can't explain the magic behind, but it solved my problem.